harvester/DBInterface.py

```diff
@@ -273,15 +281,15 @@ def delete_all_related_records(self, crosstable, record_id):
 
     def delete_one_related_record(self, crosstable, column_value, record_id):
         columnname = self.get_table_value_column(crosstable)
-        self.delete_row_generic(crosstable, columnname, column_value, "and record_id="+str(record_id) )
+        self.delete_row_generic(crosstable, columnname, column_value, "and record_id="+str(record_id))
 
     def delete_row_generic(self, tablename, columnname, column_value, extrawhere=""):
         con = self.getConnection()
         with con:
             cur = self.getCursor(con)
             try:
                 sqlstring = "DELETE from {} where {}=? {}".format(tablename, columnname, extrawhere)
-                cur.execute(self._prep(sqlstring), (column_value,))
+                cur.execute(self._prep(sqlstring), column_value)
```

Comment:
Is this change tested on both Sqlite and Postgresql?  There was a reason that the singular value was put into a tuple at the time, and I think it was because a single string value didn't work on one or the other platform with the execute() command. If the dangling comma looks bad, and if testing shows this revised code doesn't work on both, then perhaps just wrapping in list brackets would look cleaner.

Reply:
I can't seem to find where this was introduced, but it was definitely in error and breaks the harvester on postgres, I will revert it. Thanks for catching this!